.travis.yml

```diff
@@ -32,8 +32,7 @@ install:
 
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      cp -r cc_plugin_ncei/tests/ /tmp && cd /tmp ;
-      py.test -s -rxs -v tests ;
```

Comment:
This is a "safer" way to run tests b/c that is the condition similar to the user machine, where things are tested from a installed version. I removed it b/c there are a number of failures due to probably hardcoded paths in the tests.

I can fix those later, it is not a big deal.